server/app/assets/stylesheets/styles.css

```diff
@@ -29,14 +29,10 @@
     @apply bg-white text-gray-400 border border-gray-400;
   }
 
-  .cf-accordion-visible > .cf-accordion-header > .cf-accordion-button {
+  button[aria-expanded='true'] > .cf-accordion-button {
     @apply rotate-180;
```

Comment:
Does @apply actually do anything? I recall a blog post a while ago that said it was removed. Confirmed by [caniuse](https://caniuse.com/?search=%40apply).

Reply:
I didn't know this, but, apparently, this is a [Tailwind annotation](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply).